tags/environmentalsustainability.md

```diff
@@ -0,0 +1,77 @@
+# CNCF TAG Environmental Sustainability
+
+## Introduction
+
+The Paris Climate Accords outline the link between economic activity, greenhouse gas (GHG) emissions, and the impacts of climate change. GHG  emissions, primarily made from Carbon Dioxide, are released during the combustion of fossil fuels to produce electricity. To reach net zero goals, we need to identify how to reduce the environmental footprints of increasingly large data centers.
+
+## Mission Statement
+This TAG's goal is to advocate for, develop, support, and help evaluate environmental sustainability initiatives in Cloud Native Technologies.
+
+All activities of this TAG should aim to positively impact the community by changing mindset, increasing resource efficiency, and putting energy efficiency and environmental sustainability on everyone's agenda.
+
+## Responsibilities & Deliverables
+
+### Background
+We recognize that: 
+- balancing resource consumption with the desire for performance is difficult
+- there is inconsistency in how emissions are calculated and what is included by software in scopes of emission. 
+
+### In-Scope
+
+- Identify, define, and develop tooling to assess and improve environmental sustainability approaches, including
+    - quantification of energy consumption of full-stack software including application, cloud native services
+    - recommendations to manage full-stack software such that the overall energy consumption and associated carbon are reduced.
+    - items to evaluate technological and architectural health of projects
```

Comment:
What is meant by "items to evaluate technological and architectural health of projects"? Do "items" refer to tooling or something else? What is meant by "technological and architectural health"?

Reply:
tooling and/or processes-it's a generic term.  "technological and architectural health" has to do with general soundness of the projects in terms of what is needed.  An example would be the applying something like this: https://github.com/Green-Software-Foundation/software_carbon_intensity to a project, or known measurement models, et cetera.  This sort of science can be hard.

Reply:
I would change "items" to "capabilities, benchmarks, and processes" to add the nuance you are seeking.

As to health, this often has specific meaning in a variety of contexts.  We think of community health of a project, or security health of a project. I'm not opposed to environmental health being introduced in this context however when used be sure to caveat as environmental to avoid confusion.

Reply:
Will add to the PR.

Reply:
Updated PR with new wording from meeting today.  Many thanks.